lib/Xud.ts

```diff
@@ -1,15 +1,37 @@
-const Logger = require('./Logger');
-const Config = require('./Config');
-const DB = require('./db/DB');
-const OrderBook = require('./orderbook/OrderBook');
-const LndClient = require('./lndclient/LndClient');
-const RaidenClient = require('./raidenclient/RaidenClient');
-const RpcServer = require('./rpc/RpcServer');
-const P2P = require('./p2p/P2P');
-const P2PServer = require('./p2p/P2PServer');
+import Logger from './Logger';
+import Config from './Config';
+import DB from './db/DB';
+import OrderBook from './orderbook/OrderBook';
+import LndClient from './lndclient/LndClient';
+import RaidenClient from './raidenclient/RaidenClient';
+import RpcServer from './rpc/RpcServer';
+import P2P from './p2p/P2P';
+import P2PServer from './p2p/P2PServer';
+
+interface XudInterface {
```

Comment:
@sangaman , I realized adding interfaces is redundant here. Class interface can be used in case we want several classes to implement the same methods / properties. There's no reason to create a distinct interface for each class. I removed the interfaces, class properties remains on top of the class declaration, before the constructor.